projects/graphql-client/src/graphql-config.ts

```diff
@@ -30,6 +30,7 @@ export interface GraphQlQueryHandler<TRequest, TResponse> extends GraphQlHandler
 
 export interface GraphQlMutationHandler<TRequest, TResponse> extends GraphQlHandler<TRequest, TResponse> {
   readonly type: GraphQlHandlerType.Mutation;
+  reportMutation?(request: TRequest): void;
```

Comment:
Do we need to even modify the mutation handlers? The main request service knows every time it invokes a mutation, right?